ably/http/httputils.py

```diff
@@ -12,15 +14,10 @@ class HttpUtils:
     }
 
     @staticmethod
-    def default_get_headers(binary=False, variant=None):
-        if variant is not None:
-            lib_version = 'python.%s-%s' % (variant, ably.lib_version)
-        else:
-            lib_version = 'python-%s' % ably.lib_version
-
+    def default_get_headers(binary=False):
         headers = {
             "X-Ably-Version": ably.api_version,
-            "X-Ably-Lib": lib_version,
+            "Ably-Agent": 'ably-python/%s python/%s ably-python-rest' % (ably.lib_version, platform.python_version())
```

Comment:
These values will need to be added to [this file](https://github.com/ably/ably-common/blob/main/protocol/agents.json) (see https://github.com/ably/ably-common/pull/92 where I have updated the documentation for doing this, which can be viewed directly [here](https://github.com/ably/ably-common/tree/agents-docs/protocol#agents))

Reply:
https://github.com/ably/ably-common/pull/93

Reply:
This has been done. I'm taking the liberty of resolving this conversation on @lmars' behalf. 😄